staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/apiserver.go

```diff
@@ -131,13 +135,25 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
 		GenericAPIServer: genericServer,
 	}
 
+	// TODO: move this to a another file
```

Comment:
Why not do it now? Any blockers?

Reply:
Currently, only one group version in the apiserver,  I am not sure whether we need more groups in the future.